### DISCUSSION TITLE
Carla & FMI (Functional Mockup Interface)

### DISCUSSION THREAD

**Discussion by chd0043** [carla-simulator/carla#6827] [Q&A]
Someone knows if the current release (0.9.14) of Carla support FMI (Functional Mockup Interface) / FMU (Functional Mockup Unit)? 
Or has anyone tried to integrate it via the Unreal Engine FMIPlugin (https://github.com/ORNL-Modelica/UnrealEngine-FMIPlugin) or any other way (if possibe)?

**Comment by FriedrichZimmer** [comment]
I'm currently also searching for something like this, but I'm not very familar with the FMI standard. I think it should be easy to create an FMI bridge similar to the ROS bridge. Has anyone already tried this?
